src/main/java/org/mskcc/cbio/oncokb/config/application/ApplicationProperties.java

```diff
@@ -171,6 +176,10 @@ public void setFrontend(FrontendProperties frontend) {
         return getDomains(this.getTrialedDomains());
     }
 
+    public List<String> getEmbargoedCountriesList() {
+        return getDomains(this.getEmbargoedCountries());
+    }
+
     private List<String> getDomains(String domainStr) {
```

Comment:
Maybe we should rename this method now  to `stringSplitAndTrim`? And good name recommendation?

Reply:
How does `toList` sound? Or `stringToList`, `getList`